BibBuddy/app/src/main/res/layout-v26/fragment_book_notes.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/fragment_book_notes"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_marginTop="65dp">
```

Comment:
Why fragment_book_note.xml as layout and as laayout-v26?

Reply:
AndroidStudio created that by itself after changing minSDK to 26.